SDKs/_metadata/resources_resource-manager.txt

```diff
@@ -0,0 +1,14 @@
+﻿Installing AutoRest version: latest
```

Comment:
this is not the proper place for this file.  It should be in: eng\mgmt\mgmtmetadata\<rpName>.txt

Reply:
Sorry I got a bit confused with the auto generated path. Moved it to the right location.